furnace/pid1.py

```diff
@@ -181,6 +183,14 @@ def run(self):
         logger.debug("Control pipe closed, stopping")
         return 0
 
+    # NOTE: use only before create_namespaces()
+    def get_loop_devices(self):
+        LoopDevice = collections.namedtuple('Device', 'name minor')
```

Comment:
super-nitpick: the 'Device' in the namedtuple constructor should also be LoopDevice, as that's what's gonna be visible in backtraces and such.